pkg/controller/sriovnetworknodepolicy/sriovnetworknodepolicy_controller.go

```diff
@@ -570,8 +570,12 @@ func renderDevicePluginConfigData(pl *sriovnetworkv1.SriovNetworkNodePolicyList)
 		}
 
 		found, i := resourceNameInList(p.Spec.ResourceName, &rcl)
-		netDeviceSelectors := dptypes.NetDeviceSelectors{}
```

Comment:
I think we can keep this unchanged, and not declare later for each condition, can't we?

Reply:
done